src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotUseInsecureDeserializationLosFormatterTests.cs

```diff
@@ -8,7 +8,6 @@
 
 namespace Microsoft.NetCore.Analyzers.Security.UnitTests
 {
-    [Trait(Traits.DataflowAnalysis, Traits.Dataflow.PropertySetAnalysis)]
```

Comment:
Are these no longer based on flow analysis? Or are the individual tests tagged with a trait?

Reply:
These analyzers weren't DFA based.  They were just looking for invocation and method reference operations.

Reply:
Ah got it, thanks!